### DISCUSSION TITLE
Can we create submap shortcuts?

### DISCUSSION THREAD

**Discussion by txtyash** [wezterm/wezterm#2549] [Q&A] [answered]
It would save me some keys if I were able to create submaps such as:
- `Alt-t n`: newtab
- `Alt-t x`: closetab
.
.
.

**Comment by wez** [comment, marked as answer]
Take a look at:
* [Leader Key](https://wezfurlong.org/wezterm/config/keys.html#leader-key) for the simplest way
* [Key Tables](https://wezfurlong.org/wezterm/config/key-tables.html) for a more powerful way
